.github/workflows/build.yaml

```
@@ -86,54 +114,112 @@ jobs:
+          echo "SERVICE_BUILD_CONFIG: $SERVICE_BUILD_CONFIG"
+          echo "DB_BUILD_CONFIG: $DB_BUILD_CONFIG"
+          
+          # Step 7: Extract Required Environment Variables
+          SERVICE_DOCKERFILE=$(echo "$SERVICE_BUILD_CONFIG" | yq eval -r '.dockerfile // ""' -)
+          SERVICE_IMAGE_NAME=$(echo "$SERVICE_BUILD_CONFIG" | yq eval -r '.["image-name"] // ""' -)
+          SERVICE_WORK_DIR=$(echo "$SERVICE_BUILD_CONFIG" | yq eval -r '.["work-dir"] // ""' -)
+          
+          DB_IMAGE_NAME=$(echo "$DB_BUILD_CONFIG" | yq eval -r '.["image-name"] // ""' -)
+          DB_WORK_DIR=$(echo "$DB_BUILD_CONFIG" | yq eval -r '.["work-dir"] // ""' -)
+          DB_DOCKERFILE=$(echo "$DB_BUILD_CONFIG" | yq eval -r '.["work-dir"] // ""' -)/Dockerfile
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)